package maidsafe.lifestuff;

message PassportData {
  required bytes serialised_keyring = 1;
}

message DriveData {
  required bytes unique_user_id = 1;
  required bytes root_parent_id = 2;
  required int64 max_space = 3;
  required int64 used_space = 4;
}

message DataAtlas {
  optional DriveData drive_data = 1;
  required PassportData passport_data = 2;
  required bytes timestamp = 3;
}
